artemis-core-client/src/main/java/org/apache/activemq/artemis/core/remoting/impl/netty/NettyConnector.java

```diff
@@ -526,6 +526,14 @@ public synchronized void start() {
          channelClazz = KQueueSocketChannel.class;
          logger.debug("Connector " + this + " using native kqueue");
       } else {
+         if (useEpoll) {
+            ActiveMQClientLogger.LOGGER.nettyEpollNotAvailable(this);
+         }
+
+         if (useKQueue) {
+            ActiveMQClientLogger.LOGGER.nettyKQueueNotAvailable(this);
+         }
+
```

Comment:
I agree with Robbie...


These warnings are only valid if you assert the environment first (Linur for Epoll or Mac for Queue)


If the ./artemis create was setting this accordingly to the environment you're on.. then you would be ok on doing this warn... 

you just need the env asserted somehow either before throwing the log or the default CLI setting these arguments accordingly to the env you're on.

Reply:
the artemis create comment I made only applies to the acceptor. for the client you would need to validate the environment first before you throw this warning... either assert the env or make it debug.

Reply:
Even with the environment asserted, there are cases in actual use that its not going to be available by expectation and so such a log is going to be wrong/misleading. I would simply log what is actually in use (which is already done, at debug), not try and special case warnings. Simpler, and never wrong.

Reply:
for these other cases, the user should just not enable epoll... I think the warn would still be a good idea. that's just IMO though.

Reply:
Its enabled by default and gated for availability checks specifically because there are various cases it wont be available by expectation, I dont think users should have to explicitly disable an essentially optional thing to avoid a warning in such expected cases.

Reply:
I'll change it to raise the warning only if the `useEpoll` parameter is explicitly set